docs/README.Windows.md

```diff
@@ -77,40 +77,25 @@ Default options are fine.
 Start the VS2019 installer and click **Workloads** select
 * Under **Desktop & Mobile** section select
   * `Desktop development with C++`
-  * `Universal Windows Platform development` (if compiling for UWP or UWP-ARM)
 
 Click in **Individual components** select
-* Under **Compilers, build tools and runtimes** section select
-  * `MSVC v142 - VS 2019 C++ ARM build tools (Latest)` (if compiling for UWP-ARM)
 * Under **SDKs, libraries, and frameworks** section select
-  * `Windows 10 SDK (10.0.18362.0)` (if compiling for UWP or UWP-ARM)
+  * `Windows 10 SDK (10.0.18362.0)`
```

Comment:
Read my comments is not need select any additional individual component for compiling x64 desktop

Windows 10 SDK (10.0.19041.0) is fine and is always installed by default.

Reply:
Thing is. This is not when using default settings. This is specifically selecting only what is needed to build. To reduce size. If sdk isn't mentioned then I'd not select that and it won't work.